docs/user-guide/getting-started-with-docker-and-lets-encrypt.md

```diff
@@ -0,0 +1,181 @@
+
+On the Docker host, run the following command:
+
+`$ docker network create web`
+
+Now, let's create a directory on the server where we will configure the rest of Traefik:
+
+`$ mkdir -p /opt/traefik`
+
+Within this directory, we're going to create 3 empty files:
+
+```sh
+$ touch /opt/traefik/docker-compose.yml
+$ touch /opt/traefik/acme.json
+$ touch /opt/traefik/traefik.toml
+```
+
+The `docker-compose.yml` file will provide us with a simple, consistent and more importantly, a deterministic way to create Traefik. The contents of the file is as follows:
+
+```yaml
+version: '2'
+
+services:
+  traefik:
+    image: traefik:1.3.5
+    restart: always
+    ports:
+      - 80:80
+      - 443:443
+    networks:
+      - web
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /srv/traefik/traefik.toml:/traefik.toml
+      - /srv/traefik/acme.json:/acme.json
+    container_name: traefik
+
+networks:
+  web:
+    external: true
+```
+
+As you can see, we're mounting the `traefik.toml` file as well as the (empty) `acme.json` file in the container. Also, we're mounting the `/var/run/docker.sock` Docker socket in the container as well, so Traefik can listen to Docker events and reconfigure it's own internal configuration when containers are created (or shut down).
+Also, we're making sure the container is automatically restarted by the Docker engine in case of problems (or: if the server is rebooted). We're publishing the default HTTP ports `80` and `443` on the host, and making sure the container is placed within the `web` network we've created earlier on. Finally, we're giving this container a static name called `traefik`.
+
+Let's take a look at a simply `traefik.toml` configuration as well before we'll create the Traefik container:
+
+```toml
+debug = false
+checkNewVersion = true
+logLevel = "ERROR"
+defaultEntryPoints = ["https","http"]
+
+[entryPoints]
+  [entryPoints.http]
+  address = ":80"
+    [entryPoints.http.redirect]
+      entryPoint = "https"
+  [entryPoints.https]
+  address = ":443"
+  [entryPoints.https.tls]
+
+[retry]
+
+[docker]
+endpoint = "unix:///var/run/docker.sock"
+domain = "my-awesome-app.org"
+watch = true
+exposedbydefault = false
+
+[acme]
+email = "your-email-here@my-awesome-app.org"
+storage = "acme.json"
+entryPoint = "https"
+OnHostRule = true
+```
+
+This is the minimum configuration required to do the following:
+
+- Log `ERROR`-level messages (or more severe) to the console, but silence `DEBUG`-level messagse
+- Check for new versions of Traefik periodically
+- Create two entrypoints, namely an `HTTP` endpoint on port `80`, and an `HTTPS` endpoint on port `443` where all incoming traffic on port `80` will immediately get redirected to `HTTPS`.
+- Enable the Docker configuration backend and listen for container events on the Docker unix socket we've mounted earlier. However, **new containers will not be exposed by Traefik by default, we'll get into this in a bit!*
```

Comment:
A _*_ character is missing at the end of the line to terminate the **bold** section.